### PR TITLE
CPPF DQM Occupancy

### PR DESCRIPTION
**Pull request by cobby319**
#### PR description:

implement CPPF Input and Output Occupancy DQM 

#### PR validation:

test using standard Release Validation workflow: runTheMatrix.py --what standard -l 136.867
see full introduction here: https://twiki.cern.ch/twiki/bin/view/Sandbox/L1TOccupancyPlots

### CODE REVIEW

**Comment by jfernan2** [comment]
Hi @cobby319, could you please make sure that subsystem name appears in the title of the PR?

**Comment by jfernan2** [comment]
@cobby319 we don't have you identified as a L1T DQM developer, and git does not tell us your name. Can you please add yourself with your git name in the corresponding e-group? https://twiki.cern.ch/twiki/bin/viewauth/CMS/DQMContacts#L1T

File: DQM/Integration/python/config/inputsource_cfi.py [hunk 1]
```diff
@@ -67,7 +67,7 @@
         SelectEvents = cms.untracked.vstring('*'),
         streamLabel = cms.untracked.string('streamDQM'),
         scanOnce = cms.untracked.bool(options.scanOnce),
-        minEventsPerLumi = cms.untracked.int32(1),
+        minEventsPerLumi = cms.untracked.int32(-1),
```
Comment (fioriNTU) [inline DQM/Integration/python/config/inputsource_cfi.py]: @cobby319 this is a general DQM configuration file for Online, this is not supposed to be changed for subsystems use. Please revert back this change.

**Comment by fabiocos** [comment]
@cobby319 is this the backport of which PR? 10_6_X is a production cycle, new developemnts should be done first in maser (i.e. 11_0_X as of now). What is the plan for this PR? It looks like as dormant since a while.

**Comment by cobby319** [comment]
> @cobby319 is this the backport of which PR? 10_6_X is a production cycle, new developemnts should be done first in maser (i.e. 11_0_X as of now). What is the plan for this PR? It looks like as dormant since a while.

Sorry for the late reply.
It seems to be something misunderstood from my side. I think I should make it go to master first and the do the backport, which is not the case here.

**Comment by fioriNTU** [comment]
@cobby319 , please address my comment here https://github.com/cms-sw/cmssw/pull/27416#discussion_r308107945

**Comment by fabiocos** [comment]
@cobby319 any news about this PR?

**Comment by jfernan2** [comment]
@cobby319 if this PR is not valid any longer, please close it. Thanks

**Comment by fabiocos** [comment]
@cobby319 this PR looks abandoned, I close it, please reopen it when ready to work on it.

**Pull request closed without merging**